app/shared/src/datetime/__tests__/__snapshots__/Date.test.js.snap

```diff
@@ -17,7 +17,7 @@ exports[`formats date 1`] = `
     ]
   }
 >
-  2017-11-17
+  11/17/2017
```

Comment:
This test looks fragile. Will it break with a different setting? 🙂 Maybe the test is complete nonsense and we should rewrite it to actually test something (instead of snapshoting).

Reply:
Since `device-info` is mocked, I am not sure it actually returns anything, so moment falls back to default locale. I do think you are correct, this test really does not test anything right now

Reply:
> this test really does not test anything right now

@tbergq Get rid of it.